src/app/App.tsx

```diff
@@ -1,50 +1,61 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { I18nextProvider } from 'react-i18next';
-import { ApolloClient, InMemoryCache, ApolloProvider } from '@apollo/client';
 import '@patternfly/react-core/dist/styles/base.css';
 import { AppLayout, Loading } from '@app/components';
 import { AppRoutes } from '@app/routes';
 import { initI18N } from '@i18n/i18n';
 import { KeycloakAuthProvider, KeycloakContext, getKeycloakInstance } from '@app/auth/keycloak';
 import '@app/app.css';
+import { useVerifyTokenQuery } from '@app/models';
 
 let keycloak: Keycloak.KeycloakInstance | undefined;
+let tokenresponse: string | boolean | unknown;
 
 const App: React.FunctionComponent = () => {
   const [initialized, setInitialized] = useState(false);
-  const client = new ApolloClient({
-    uri: process.env.GRAPHQL_ENDPOINT || 'http://localhost:8080/graphql',
-    cache: new InMemoryCache(),
+  const location = window.location;
```

Comment:
The simplest way to get the params in react is hook `useParams`. Lets take the advantage of react hooks. 
import { useParams } from "react-router-dom";
const {token, email}=useParams<{token:string, email:string}>();

Reply:
Used `useLocation` instead of `useParams`